cms/models.py

```diff
@@ -30,15 +30,7 @@ class HomePage(Page):
     """
     CMS page representing the homepage.
     """
-    title_background = models.ForeignKey(
-        Image,
-        null=True,
-        blank=True,
-        on_delete=models.SET_NULL,
-    )
-    content_panels = Page.content_panels + [
-        FieldPanel('title_background')
-    ]
+    content_panels = []
```

Comment:
@singingwolfboy if i remove this link then it will show ``Title`` in cms. Because it inherits title from parent. And we want to remove ``title`` along with ``title_background``.